internal/pkg/template/templates/environment/cf.yml

```diff
@@ -646,6 +646,21 @@ Resources:
                   - logs:DescribeLogStreams
                 Resource: "*"
 {{- end}}
+{{- if .Addons}}
+  AddonsStack:
+    Metadata:
+      'aws:copilot:description': 'An addons CloudFormation stack for your additional AWS resources'
```

Comment:
Maybe
```suggestion
      'aws:copilot:description': 'A CloudFormation nested stack stack for your addons resources'
```
?

Reply:
I updated it to `'A CloudFormation nested stack for your additional AWS resources'`!